luaext/Makefile.am

```diff
@@ -5,8 +5,8 @@ EXTRA_DIST = \
 
 AM_CPPFLAGS = -I$(top_builddir) -I$(top_srcdir) -I$(top_builddir)/include -I.
 
-libluaext_la_CPPFLAGS = $(AM_CPPFLAGS) -DWITH_POSIX @LUA_CFLAGS@
-libluaext_la_LIBADD = @LUA_LIBS@
+libluaext_la_CPPFLAGS = $(AM_CPPFLAGS) -DWITH_POSIX $(LUA_CFLAGS)
+libluaext_la_LIBADD = $(LUA_LIBS)
 libluaext_la_SOURCES = \
```

Comment:
What's with this change? The rest of the code still uses the @LUA_XXX@ syntax, so why is this done here?
(ZSTD_XXX is the other odd candidate here)

Reply:
Oh, good catch. I initially looked at going the ZSTD-route, but then switched back but this one case was left behind.

Reply:
This change dropped in the last push.